administrator/language/en-GB/en-GB.plg_system_shortcut.ini

```
@@ -0,0 +1,47 @@
+PLG_SYSTEM_SHORTCUT="System - Keyboard Shortcut"
+PLG_SYSTEM_SHORTCUT_XML_DESCRIPTION="System - Keyboard Shortcut Plugin"
+PLG_SYSTEM_SHORTCUT_TRUE = "True"
```

```suggestion
PLG_SYSTEM_SHORTCUT_TRUE="True"
```

There should not be any spaces

---

Yes, I removed it.